src/css/base.scss

```diff
@@ -314,10 +314,32 @@ td {
   font-size: 20px;
 }
 
+.daily-double-wage,
+.main__game__all-wagers,
+.main__game__all-answers,
 .daily-double-wager {
   display: none;
 }
 
 .reveal {
   display: block;
-}
\ No newline at end of file
+}
+
+.header-final-round {
+  height: 80px;
+  color: $yellow;
```

Comment:
With color variables, try to find a name for the variable for what it represents and not the color. What if the color changes from yellow to orange? Then the variable name has to change as well.

Reply:
@robbiejaeger great point Robbie! I did not think about that :D